recipes/pyflamegpu/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "pyflamegpu" %}
+{% set version_py = "2.0.0rc1" %}
+{% set version_cpp = "2.0.0-rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version_py }}
+
+source:
+  # Temporary, will switch to release, when conda recipe is working
+  url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/heads/rob_conda.tar.gz
+  # url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/tags/{{ version_cpp }}.tar.gz
+
+build:
+  number: 0
+  script_env:
```

```suggestion
build:
  number: 0
  skip: true  # [cuda_compiler_version == "None"]
  skip: true  # [py < 310]
  script_env:
```
Skip building the default non-cuda variant forever. Skip lower python versions for faster debugging for now, then unskip later.